src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,40 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val englishGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
```

Comment:
FYI such simple method could be single-lined.
And please consider use constants or resources instead of hardcoded strings